change/@azure-msal-angular-5ff472d7-ca65-4dfd-8dea-a5388d6d930d.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
These should all be none changes as this doesn't affect the published packages

Reply:
Yeah meant to update this, thanks for catching it!